README.md

```diff
@@ -67,15 +67,12 @@ Then type path and name of new configuration file.
 
 To create new stack you have to type:
 
-``~ $ perun --mode=create-stack
-             --template=<PATH TO YOUR TEMPLATE>
-             --stack=<NAME OF YOUR STACK>
+``~ $ perun create-stack <PATH TO YOUR TEMPLATE> <NAME OF YOUR STACK>
 ``
 
 To destroy stack just type:
 
-``~ $ perun --mode=delete-stack
-             --stack=<NAME OF YOUR STACK>
+``~ $ perun delete-stack <NAME OF YOUR STACK>
 ``
 
 ### Configuration file
```

Comment:
I'm missing documentation for `setup-remote-sink` and `destroy-remote-sink`, including what it will create and what are the costs for this solution.